scripts/start_cr_server.sh

```diff
@@ -2,76 +2,116 @@
 # (c) Crowdrender 2019 - 2022
 
 function download_cr {
-    if test -z $token; then
+
+
+    # exit early if a docker bind mount is specified, but no version number is provided.
+    if [ -z "$local_cr_path" ] && [ -z "$cr_version" ]; then
```

Comment:
I don't quite understand this. 

If I understood this correctly, this will only be true if someone manually sets `-e local_cr_path=""` which seems unlikely since default is `true`.

Is `local_cr_path` inside crowdrender?

If not I think it would make more sense to set default to `false` and then check against `true`.